integration/kubernetes/overhead/test_nginx/check_before_get_overhead.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+max_tries="180"
+try=1
+
+while ! kubectl logs ab | grep "HTTP/1.1 200 OK" >/dev/null; do
+	echo "wait until ab can request to server"
+	sleep 1
```

Comment:
why we have a sleep of 1 second? does it change in different systems?

Reply:
not really I wanted to do in intensive polling until I get what I want so 1 second is good to keep polling I think.